src/Compilers/VisualBasic/Portable/Syntax/Syntax.xml

```diff
@@ -5098,7 +5098,7 @@ Unify Enum declarations with other block type declarations.
         <description>The "As" keyword.</description>
       </child>
             
-      <child name="Type" kind="@TypeSyntax">
+      <child name="Type" optional="true" kind="@TypeSyntax">
         <description>The type-name part of the tuple element syntax.</description>
       </child>
```

Comment:
I see, thanks for pointing this out. With this change we end up with the node that has all its members optional, however language doesn't allow all three pieces to be omitted at the same type. Please correct me if I am wrong, I believe that language supports these two forms:
- <Identifier> [As <type>] 
- <type>

I think we should consider structuring syntax model accordingly and have a dedicated derived node for each unique syntactic form. 

---
In reply to: [82036116](https://github.com/dotnet/roslyn/pull/14284#discussion_r82036116) [](ancestors = 82036116,82035049)